source/game/kart/KartMove.cc

```diff
@@ -35,4 +36,21 @@ void KartMove::setInitialPhysicsValues(const EGG::Vector3f &pos, const EGG::Vect
     sub()->initPhysicsValues();
 }
 
+void KartMove::setKartSpeedLimit() {
+    constexpr f32 limit = 120.0f;
+    m_hardSpeedLimit = limit;
+}
```

Comment:
This function is so simple and easy to understand that it might not even be worth using `constexpr`.

Reply:
Up to you, given you work with `constexpr` more than I have.